test/PlatformFunctions.spec.ts

```diff
@@ -181,5 +184,59 @@ for (const platform in platforms) {
         expect(await platformFunctions.getUname()).to.be.null;
       });
     });
+
+    describe('createEmitter', () => {
+      // Only run tests if EventTarget is available (Node >= 15).
+      if (!process.version.startsWith('14')) {
+        let oldCreateEmitter: StripeEmitter;
+
+        if (!isNodeEnvironment) {
+          before(() => {
+            oldCreateEmitter = platformFunctions.createEmitter;
+
+            // StripeEmitter uses the Event Web API. `createEmitter` needs
+            // to be defined here to avoid reference errors in Node versions <15.
+            platformFunctions.createEmitter = (): StripeEmitter => {
+              return new StripeEmitter(new EventTarget());
+            };
```

Comment:
Is there a better way to share this `createEmitter` implementation with `stripe.worker.ts` without defining it in its own module? Should I just create a new module?